test_gender_selection.py

```diff
@@ -30,15 +31,17 @@ def test_loading_menu_header_select(self, driver):
         page_one.open_page()
         check_text_equal(page_one.element_loading_menu_select().get_text(), page_one.select_text)
 
-    def test_decals_head_1_male(self, driver):
+    @pytest.mark.parametrize("genus", ['male', 'female'])
```

Comment:
Let's make it more complicated, but pretty! 
Pass field name as param and then use gettattr to obtain object :)